FOSSAsia.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,13 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		1884C8DC20D639F8004B221F /* SignupVcMethods.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1884C8DB20D639F8004B221F /* SignupVcMethods.swift */; };
+		1884C8DF20D642E6004B221F /* LoginViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1884C8DE20D642E6004B221F /* LoginViewController.swift */; };
+		1884C8E120D6430B004B221F /* LoginVCMethods.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1884C8E020D6430B004B221F /* LoginVCMethods.swift */; };
+		18D828B120D68A0600159EC0 /* Bordered Button.swift in Sources */ = {isa = PBXBuildFile; fileRef = 18D828B020D68A0600159EC0 /* Bordered Button.swift */; };
```

Comment:
Remove it!

Reply:
okk @imjog !! but Signup screen is in Scroll View only!!

Reply:
I can't able to scroll it

Reply:
Okk @imjog I will see to it then!